packages/core/src/sync-store/sqlite/store.ts

```diff
@@ -56,13 +62,15 @@ export class SqliteSyncStore implements SyncStore {
     logFilter,
     block: rpcBlock,
     transactions: rpcTransactions,
+    transactionReceipts: rpcTransactionReceipts,
     logs: rpcLogs,
     interval,
   }: {
     chainId: number;
     logFilter: LogFilterCriteria;
     block: RpcBlock;
     transactions: RpcTransaction[];
+    transactionReceipts?: RpcTransactionReceipt[];
```

Comment:
I think I'd prefer this to only have one falsey state (empty array) just like the other options.

Reply:
Good call